dlhub_sdk/tests/test_dlhub_client.py

```diff
@@ -93,20 +95,17 @@ def test_run(dl):
 
 
 def test_submit(dl, mocker):  # noqa: F811 (flake8 does not understand usage)
-    # Make an example function
-    model = PythonStaticMethodModel.create_model('numpy.linalg', 'norm')
-    model.dlhub.test = True
-    model.set_name('1d_norm')
-    model.set_title('Norm of a 1D Array')
-    model.set_inputs('ndarray', 'Array to be normed', shape=[None])
-    model.set_outputs('number', 'Norm of the array')
 
+    # patch build_container, register_funcx, and search_ingest
```

Comment:
test that does more than just mock (just a very simple test that uses whatever your manual test code is now)